lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/model/BucketCache.java

```diff
@@ -55,10 +55,11 @@
      */
     public BucketCache(final int numberBuckets,
             final Graph<SequenceSegment> graph) {
-        this.numberBuckets = numberBuckets;
+        // Number of buckets is ceiled to a power of 2. Needed for diagram view.
+        this.numberBuckets = (int) Math.pow(2, Math.ceil(Math.log(numberBuckets) / Math.log(2)));
```

Comment:
When the result of `Math.pow` is 15.9999999999999999, `numberBuckets` will become 15.